js/apps/system/_api/gharial/APP/gharial.js

```diff
@@ -745,7 +745,7 @@ router.post('/:graph/edge/:collection', function (req, res) {
   // check existence of _from and _to vertices
   // _from vertex
   try {
-    db._document(req.body._from);
+    db._exists(req.body._from);
```

Comment:
According to the documents, this method would only return error when it got bad parameter,otherwise its return a boolean value.

> Instead of returning the found document or an error, this method will only return an object with the attributes _id, _key and _rev, or false if no document with the given _id or _key exists. It can thus be used for easy existence checks.

Reply:
:+1: _exists is different than _document